smdebug/tensorflow/base_hook.py

```diff
@@ -449,6 +454,12 @@ def _make_numpy_array(tensor_value):
         Convert the tensor value into a numpy array.
         Here it's already numpy array
         """
+        if is_tf_version_2x() and tf.executing_eagerly():
+            if (
```

Comment:
else?
Wht if statement is necessary, and what are the cases where this if statement wont be executed?

Reply:
this if statement wont be executing when eager execution is disabled with TF 2.x. Else, non-eager execution of TF 2.x and non-eager execution of TF 1.x will use the make_numpy_array() that is being used by all frameworks (tfevent/util.py)